portfolio/src/app/home/banner/banner.component.ts

```diff
@@ -9,7 +9,71 @@ export class BannerComponent implements OnInit {
 
   constructor() { }
 
+  interval: any;
+
+
   ngOnInit(): void {
+
+    this.resetAnimations();
+    this.changeBannerText();
+    this.isItBigScreen();
+
+  }
+
+  ngOnDestroy(): void {
+    clearInterval(this.interval);
+  }
+
+  isItBigScreen(): boolean {
+    let rootElement = document.querySelector(':root') as HTMLElement;
+    let shadow = document.getElementById("shadowText") as HTMLElement;
+    if(window.innerWidth < 1500) {
```

Comment:
Please place number 1500 in const variable and give it a name "WIDE_SCREEEN_SIZE". For example 
const WIDE_SCREEN_SIZE = 1500 and use it in if statement like this 
if (widnow.innerWidth < WIDE_SCREEEN_SIZE)